src/go/rpk/pkg/cli/cmd/container/start.go

```diff
@@ -304,6 +321,43 @@ func startNode(
 	return err
 }
 
+func checkBrokers(nodes []node) func() error {
+	return func() error {
+		addrs := make([]string, 0, len(nodes))
+		for _, n := range nodes {
+			addrs = append(addrs, n.addr)
+		}
+		client, err := kafka.InitClient(addrs...)
+		if err != nil {
+			return err
+		}
+		lenBrokers := len(client.Brokers())
+		if lenBrokers != len(nodes) {
+			return fmt.Errorf(
+				"Expected %d nodes, got %d.",
+				len(nodes),
+				lenBrokers,
+			)
+		}
+		return nil
+	}
+}
+
+func waitForCluster(check func() error) error {
+	retries := uint(3)
+	return retry.Do(
+		check,
+		retry.Attempts(retries),
+		retry.DelayType(retry.FixedDelay),
+		retry.Delay(1*time.Second),
```

Comment:
are 3 seconds enough? 🤔

Reply:
Not enough indeed! I was just testing in an older Mac and it timed out, so I increased the # of retries to 10.

Reply:
optionally it would be nice if this was configurable

Reply:
Done!